ci-operator/step-registry/ipi/conf/aws/usage-info/ipi-conf-aws-usage-info-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: ipi-conf-aws-usage-info
+  from_image:
+    namespace: ocp
+    name: "4.16"
```

Comment:
should this be hard-coded as `4.16`?

Reply:
Yeah, the step does not care about the version of image.